package.json

```diff
@@ -142,9 +151,6 @@
     }
   },
   "lint-staged": {
-    "*.{ts,html,scss,md,yml}": [
-      "prettier --write",
-      "git add"
```

Comment:
Does lint-staged automatically stage git files now?

Reply:
Don't really know :) It was whining that some hook uses `git add` :man_shrugging: